packages/admin-next/dashboard/src/routes/product-variants/product-variant-detail/components/variant-prices-section/variant-prices-section.tsx

```diff
@@ -0,0 +1,82 @@
+import { useTranslation } from "react-i18next"
+import { useState } from "react"
+
+import { CurrencyDollar } from "@medusajs/icons"
+import { Button, Container, Heading } from "@medusajs/ui"
+import { MoneyAmountDTO, ProductVariantDTO } from "@medusajs/types"
+
+import { ActionMenu } from "../../../../../components/common/action-menu"
+import { getLocaleAmount } from "../../../../../lib/money-amount-helpers"
+import { NoRecords } from "../../../../../components/common/empty-table-content"
+
+type VariantPricesSectionProps = {
+  variant: ProductVariantDTO & { prices: MoneyAmountDTO[] }
+}
+
+export function VariantPricesSection({ variant }: VariantPricesSectionProps) {
+  const { t } = useTranslation()
+
+  const prices = variant.prices
+    .filter((p) => !p.rules?.length)
+    .sort((p1, p2) => p1.currency_code?.localeCompare(p2.currency_code)) // display just currency prices
+
+  const [current, setCurrent] = useState(Math.min(prices.length, 3))
+
+  const hasPrices = !!prices.length
+
+  const displayPrices = prices.slice(0, current)
+
+  const onShowMore = () => {
+    setCurrent(Math.min(current + 3, prices.length))
+  }
+
+  return (
+    <Container className="flex flex-col divide-y p-0">
+      <div className="flex items-center justify-between px-6 py-4">
+        <Heading level="h2">{t("labels.prices")}</Heading>
+        <ActionMenu
+          groups={[
+            {
+              actions: [
+                {
+                  label: t("actions.edit"),
+                  to: `/products/${variant.product_id}/prices`,
+                  icon: <CurrencyDollar />,
+                },
```

Comment:
Is it possible to have the pricing editor route registered on two different paths?

Right now, if I enter the variant details page and click "Edit prices", I am returned to the product page, which is not the ideal UX. Ideally, you would stay on the page. Same thing if you click "Edit" to edit the general variant details.

I don't know if this is possible, but I wanted to ask. If it is, I think we should add that in. It makes for a slightly better user experience. 

cc @kasperkristensen

Reply:
Yes, this can be achieved but we need to modify edit routes. Will tackle this tomorrow. But wouldn't we want to have edit variant modals only on variant details page now that we have one?

Reply:
Yeah, that could make sense. So editing can only take place on the variants pages. 

However, the prices editor still need to be accessible from the product page right?

Reply:
yes 👍